### PR TITLE
Patch v6 fields, ID and examples

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Updated all fields directly from the API, with minor corrections to formatting as needed. 

Also resolves: #1570  #1565  
Change-type: patch
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci retest

File: config/dictionaries/resource.json [hunk 1]
```diff
@@ -592,10 +590,10 @@
...
     "name": "Fleet environment variable",
     "fields": [
-      "id",
-      "created_at",
-      "application",
-      "name",
+      "id", 
+      "created_at", 
+      "application", 
+      "name",
```
Comment (dfunckt) [inline config/dictionaries/resource.json:596]: There seems to be trailing space on these lines.

File: config/dictionaries/resource.json [hunk 2]
```diff
@@ -928,7 +922,9 @@
     "fields": [
       "id",
       "actor",
-      "username"
+      "username",
+      "created_at",
+      "has_legacy_link_to__organization"
```
Comment (dfunckt) [inline config/dictionaries/resource.json:927]: Let's drop this one.

Comment (vipulgupta2048) [reply config/dictionaries/resource.json:927]: Dropping `has_legacy_link_to__organization`

File: config/dictionaries/resource.json (hunk 1 above)
Comment (vipulgupta2048) [reply config/dictionaries/resource.json:596]: Trimed

File: config/dictionaries/resource.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
     "name": "Application",
     "fields": [
       "id",
+      "uuid",
```
Comment (vipulgupta2048) [inline config/dictionaries/resource.json:7]: Added the new uuid in the fields as the PR was already open

**Comment by vipulgupta2048** [comment]
@dfunckt Good to go? One final look if you can.

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Review by dfunckt — COMMENTED** [review]
Nice one @vipulgupta2048 -- merge at will.

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Pull request merged**